packages/cognito-custom-mail-lambda/src/mailer/templates/admin-user-invite.mjml

```diff
@@ -14,9 +14,12 @@
UGhjGsOa4uiIbKT5BNWLUGtUocb4n3PvMCWhfdqXkYZOhDpDFYrH96e/8OKuNSDHmoqESwreQvF6QMkjyknaN6IDkrIWCmwTyz5cktDyHts7OiAp67I5u4QvCCYk2HKsqUh0LFQEKRTc3oh9IMGatyOLpljXh0LTOI7XCaRQDAOUdJy3vWp9RQdlGYbhxR5LwxCFYhmgeG4/6g4Lnxwv67uAFYplgJL+WP/RCxWiCgW2G3ZfFiR4jl6oZSOhvnx9qudb3BWf3/zhw0c+DRMilPfohYpoT6rl91vlLbc3bz4EHJAPUd5TOfgEVYWF8p7rTx8DeEQo76laN0nEjxUIN28CcEQo76laNpqvUwXENXjmJ0J5T9VqFCrJFRjYiZ8I5T2thKoB3k6sgpSsEvOeWOlIDPemIHkWwCJlc++xF6qES/fwjZKBKO/RC6XzCaa4t82nABTp5fOe2PYNBewTrlEDPcGBhF6onNv7CTzrk8183hPbrRCLFDiQtT5Ze+Q/sbIJUYlC5xbxe5QsY/af2GrLJVR7xG5uAjykau4/JqEes0z4QJM+KfL5T22xRCxhEKCUug7gkJqE/xiDTUpbhIoFcL3nr0nwnqj5FrEEsIOPRYiSTj4AzEIV5daziwloryj5rAuAWaiQYYkPstD3ngT/MQpVJHSX54oLtwEUkvEhEDV+QtWKDVhlCVkbi0DUuMgHO7UB3iMrbRIQmIRK6F8UH6DqfLI1FoLImPCxfUL9JMBBShIY6IVKS/qXtWIEUIesBCgMQqXjjGkfH15VQgIUCFqhQn6ThahCSYACQSdUmhDbRj6wwV0JUCiE2qYjEcoTViRgoPVjKUL5wQsSQND7UYlQPvBemiRg0PuRi1A+IJO6OBj8qEWo/pFVEkCEpmEoEcoKGdQdKBdKT9iDUEWa/+RxobYYolDygIIiVwaqToUqwvp5SRvKqr64a9UAhZqKT1AYhUq7E6pYVrRF9fcPDlAomXvHIlcmoo6ESuuSdpLEqdowPKGk5QgMs1B5J0IVNWmINpnf4IQSn9AwC3XWhVDLkrQkv392YEJNxSc4vHhDpRVtMNzzHZZQP9i7e9w2YiCAwjMFrZCVsCbUCAoS7EYpFBkI3OQKuf+FgsRdiiW53jF/9L4TuODDWiPt8Mwv+PqTDMrZB7V4yeCD6kMFxby8R8mgvHlQLkoWvzxUUD/YZN6jY/Uvdp3kisvjBHXmfuo+LaX3QznDntJFPUpQn/l3r1MhNZIwDmqJUsCHhwiKx1O/gq66GwcVvBS5TeMHdf7Kp6d+rQcVonFQVyl0GT0ocurbelBObINyUuw4dFDk1Lv1oLxtUMFLMT9uUOdf/BK2e6HmTom7bHAYM6jnly88nAawHtRsGlSQLeI0WlDPP1+eGJMPYj2oyXQocRep/Ij6/VTb92+kNJRQOoVzFR5QZvd/HATY1aSrjoZBOdnoSFBolSZEu6BOstGFoNAqTTiYBTXJVpGg0CpNmMyCepXNFoJCo7TaGrGLbDYTFBpVuqNFXPWPUCKfCAqN0qRoFFSUzW4EhUYVHzpXfSYh4gkKjdJae/mCvANBoVGa5G2COso7TASFJkVNcdJgUIGg0CSvKZ6ggPygaq0RW/gMhfH4jAcUQwkgk894QDE2B/Y5msGbBaVeNjsRFNrkUwfOLqirSPX3NwgK+/LptXwN/jj2laDQJl88iXNaf24eCAptWg9qsgxqitV/G0tQ2NkpOTjgFXggWwylm12dVv+fLxAUWuVD4VUzTncTZZMTa8TQLh/Krup01Y9z/b8AKC/qkDjOtcYSXgkKLfOh5Ky5DzvPrf4BQPn16vePOM+Tl2JXJSg0Lv5f1HSSRFDVBn2BoNC8uCTG5UZB6UUKzUpQ6IBL9GQU1HQr3chHUOjCvPL1U2ZQ9reC+omg0IlD3i0zTvcVYtE8kqDQi0PWGXO6s8UX9ERQ6Mch54g53Vvw+T0RFDpymO5iEFRKuEmG66SqBIWeRMkKqsLBjrP+RVAYjFMLwcuqU9B/CAqDcWrDrSTlX/UNQWE0To1ccr4jIygMxiyoa8477wSFwZgFdcv5hTlBYTBmQSXeKTRAUKjPKqiQdTkAQWEwVkEds16CIigMxiqoWVYwNseorIK6ZL1bQlAYjFVQ16xdsQSFwVgFdcvazEJQGIxVULLGqwWCQn1GQYW8S3UJCoMxCuqYtzyMoDAYo6DmvDsLCQqDMQrqkreTiaAwGKOgrnl3rBEUBmMU1C1vozlBYTBGQck6rwYICvXZBBUkQd8QFAZjE9RREljSgjHZBDWLZM3NCeoPe3es8yYMBWD0ekD8MKEIZWNKlCXq1Pd/tyqVurWFDpbt23OeIZ9wrg0mmTpBHRfvMhAUydQJ6hVxaW4uKJKpE9T74k2ggiKZOkFFNJibC4r2qgS1xqlb+RAUyVQJart6WbWgSKZKUI+Ia3NzQZFMlaCOOOU6G1KqEtTr6v3vgiKZKkG9I67NzQVFMlWCimgxNxcU7dUIao0LyoegSKZmUOdzc0GRTI2gbnFuLx+CIpmpVHA3lOA/5aQE9B7U+S97Lj8JimSmUsW0xF8sPnRJUlOpZJv+aCu/CIpkppKIoPhHghIUPRMUCEpQ9ElQIChB0SdBgaB+7whoay6JvALa+iqJ3APaepdE9oC29pLHGtDas6TxPaC1R0njW0Br95LGEtDcrSSxBbSXZifKio8eLCWH1dCcLmwlBQf56EOOscRqJEEnUhyQ9YCiF3uCQZ9TEvTjKMOz4KMjwy/6LPjoyT74iT6n+OjLspaB2YKiNyMXZWJOf8Yt6qknOjRqUZv1Hl3ah3w1ynyPbn0N95BafZeFji1jbUjdZss9+jZQUnJiBMs0xMLvKSdGcX/0/Y7UbZv9d2Is79cxTx2aj5d9JwAAAAAAAAAAAAAAAADgR3twSAAAAAAg6P9rTxgBAAAAAAAAAAAAAAAAAAAAAAAARgG/d9nPbPi/vAAAAABJRU5ErkJggg==" />
```

Comment:
You might get a problem with this image, I think there is a max size on emails in Cognito, hence I am serving the image from a bucket previously. It might be fine, just needs testing